doc/src/manual/interfaces.md

```diff
@@ -468,10 +468,17 @@ Not all types support `axes` and indexing, but many are convenient to allow in b
 The [`Base.broadcastable`](@ref) function is called on each argument to broadcast, allowing
 it to return something different that supports `axes` and indexing. By
 default, this is the identity function for all `AbstractArray`s and `Number`s — they already
-support `axes` and indexing. For a handful of other types (including but not limited to
-types themselves, functions, special singletons like [`missing`](@ref) and [`nothing`](@ref), and dates),
-`Base.broadcastable` returns the argument wrapped in a `Ref` to act as a 0-dimensional
-"scalar" for the purposes of broadcasting. Custom types can similarly specialize
+support `axes` and indexing.
+
+If a type does *not* act like an indexable container, then it should typically be defined to act as a
+0-dimensional "scalar" for the purposes of broadcasting.  This is accomplished by defining a method:
```

Comment:
Triage wants to suggest only adding this method if a type is explicitly intended to act like a scalar. Adding it for everything that's not indexable seems like a much larger set of types.

Reply:
Okay, done.